src/charm.py

```diff
@@ -31,6 +28,10 @@
 logger = logging.getLogger(__name__)
 
 CONFIG_PATH = "/etc/agent/agent.yaml"
+METRICS_RULES_SRC_PATH = "./src/prometheus_alert_rules"
+METRICS_RULES_PATH = "./prometheus_alert_rules"
```

Comment:
We have two different paths for backwards compat?

Reply:
One path is the one where alert rules for this charm are defined. The other is where we put rules to send to prometheus. Those rules can come from this charm or one related to us.

Reply:
Perhaps add some comments about this? Currently it isn't kind to our future selves.

Reply:
I don't personally find the code that hard to follow (at least not hard enough for comments), but making it `METRICS_RULES_DEST_PATH` would pair up well.